mysite/models.py

```diff
@@ -64,3 +64,28 @@ def __str__(self):
         return self.title
 
 # Create your models here.
+#Udaan model
+class Udaan_static(models.Model):
```

Comment:
@PiyushPI07 Just a suggestion, Instead of creating a migration, we could have actually filtered data such as images and texts, from existing models like IMAGES and BLOG based on an additional fields such as page ID and referenced it on Admin panel for convenience. Separate Migrations are not created for each page generally in practice. 
@SaurabhAgarwala Your thoughts? Should we merge the migrations to schema and keep components for the Udaan Page separately in the db?

Reply:
I guess its not only about images, we still need separate models for other data such as event name and description. Its better if we follow 3rd Normal form instead of having a big image database which would not be easy for non developer admins to manage.

Reply:
For different parts of the website, it's always nice to have it as sub-modular as possible. This leads to easy maintenance. So, we will be making different apps for different components of the website. Now specifically for the Udaan page, since we have two kinds of images _events_ and _carousel_ so for that two different models is fine, but for the background image I suggest to directly add it as it will always be a single image unlike carousel and events